### ISSUE TITLE
Incomplete documentation: optimiser for large datasets

### ISSUE THREAD

**Issue by kdutia** [Accenture/AmpliGraph#215]
### Description

I'm trying to train a TransE model with a relatively large graph (1.2M triples, 600k entities) and am getting the following error when trying to run with the Adam optimizer. 

```
This mode works well only with SGD optimizer with decay (read docs for details).
Kindly change the optimizer and restart the experiment
``` 

I can't find the part of the documentation which tells me why this is the case, especially given that YAGO3-10 seems to be of a size also above the large graph threshold and that is trained on Adam in the benchmarks. 

### Actual Behavior

I am trying to train on a large model and can only use the SGD optimizer. 

### Expected Behavior

I can use another optimizer or the documentation tells me how/why to use SGD for large datasets.

Thanks

**Comment by sumitpai** [comment]
Hi @kdutia,
 [Here is a brief description about large graph mode](https://docs.ampligraph.org/en/1.3.1/dev_notes.html#dealing-with-large-graphs). Yago 3-10 has approx 123k entities, so it runs normally. We automatically switch to large graph mode when number of entities are >500k, otherwise the entity embeddings may not fit on the GPU. In this mode, only SGD optimizer is currently supported. You can bypass it by doing the following:
```
from ampligraph.latent_features import set_entity_threshold
set_entity_threshold(700000)
```

**Comment by kdutia** [comment]
Hi @sumitpai, thanks. This is exactly what I was looking for. Could I suggest pointing to that documentation page in the error message?

**Comment by sumitpai** [comment]
Okay, we can update the error msg to include a link to the doc. Will do it in upcoming release

**Comment by sumitpai** [comment]
Fixed by issue #216 and pull request #217
